src/components/pages/Offers/Offer/OfferPreviewLink/OfferPreviewLink.jsx

```diff
@@ -5,16 +5,18 @@
 import PropTypes from 'prop-types'
 import React, { useCallback } from 'react'
 
-import { WEBAPP_URL } from 'utils/config'
+import { WEBAPP_URL_NEW, WEBAPP_URL_OLD } from 'utils/config'
 
-const getOfferPreviewUrl = (offerId, mediationId) => {
-  const webappOfferUrl = `${WEBAPP_URL}/offre/details/${offerId}`
+const getOfferPreviewUrl = (offerId, mediationId, webAppV2Enabled) => {
+
+  let webappOfferUrl = ''
+  webAppV2Enabled === true ?  webappOfferUrl = `${WEBAPP_URL_NEW}/offre/details/${offerId}` : webappOfferUrl = `${WEBAPP_URL_OLD}/offre/details/${offerId}`
 
```

Comment:
```suggestion
  const webappOfferBalseUrl = webAppV2Enabled ?  WEBAPP_V2_URL : WEBAPP_URL
  const webappOfferUrl = `${webappOfferBalseUrl}/offre/details/${offerId}`
```

Voir directement donner une `baseUrl` en argument de la fonction.

Reply:
avec la v2 les url c'est {urlv2}/offre/:id et pour la description {urlv2}/offre/:id/description

Reply:
Plutot garder un argument `isWebappV2` et avoir toute la logique ici alors.